examples/carousel/carousel-2-tablist.html

```diff
@@ -0,0 +1,799 @@
+                  </div>   <!-- .item -->
+
+                  <div class="carousel-item"
+                       id="carousel-item-6"
+                       role="tabpanel"
+                       aria-roledescription="slide"
+                       aria-label="6 of 6">
+
+                    <div class="carousel-image">
+                      <a href="#!" id="carousel-image-6">
+                        <img src="images/mag800-2__800x600.jpg" alt="Mid-American Gardener panelists on the set" />
+                      </a>
+                    </div>
+
+                    <div class="carousel-caption">
+
+                      <h3>
+                        <a href="#!">
+                          Mid-American Gardener: Thursdays at 7 pm
+                        </a>
+                      </h3>
+
+                      <div class="hidden-xs hidden-sm">
+                        <p><span class="contrast">Watch the latest episodes.</span></p>
+                      </div>
+                    </div>  <!-- .carousel-caption -->
+                  </div>   <!-- .item -->
+
+                </div>
+
+
+              </div> <!-- carousel-inner -->
+
+
+            </section><!-- /.carousel -->
+
+            <div class="col-sm-1"></div>
+
+          </div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+      </section>
+
+      <section>
+        <h2>Accessibility Features</h2>
+        <h3>Controlling Automatic Slide Rotation</h3>
+        <p>
+          Users can stop and start slide rotation, which is an essential aspect of accessibility of the carousel for a variety of people with disabilities.
+          People with low vision or a cognitive disability that affects visual processing or reading benefit from being able to control slide rotation so they have sufficient time to explore slide content.
+          Similarly, since screen reader users cannot perceive automatic rotation, it can make reading the page confusing and disorienting.
+          For example, when slides are automatically rotating, a screen reader user may read an element on slide one, execute a screen reader command to read the next element, and, instead of hearing the next element on slide one, hear an element from slide 2 without any knowledge that the element just announced is from an entirely new context.
+        </p>
+        <p>This example includes the following features for giving users control over slide rotation:</p>
+        <ul>
+          <li>
+            Hovering the mouse over any carousel content pauses automatic rotation.
+            Automatic rotation resumes when the mouse moves away from the carousel unless another condition that prevents rotation, such as keyboard focus, has been triggered.
+          </li>
+          <li>
+            Moving keyboard focus to any of the carousel content, including the next and previous slide elements, pauses automatic rotation.
```

Comment:
This example does not have next and previous slide elements.
So either need to remove [**, including the next and previous slide elements,**]
or replace it with something else, such as [**, including the tab elements,**]

```suggestion
            Moving keyboard focus to any of the carousel content, including the tab elements, pauses automatic rotation.
```